tutor/src/components/tutor-popover.cjsx

```diff
@@ -51,7 +51,6 @@ TutorPopover = React.createClass
     content.querySelectorAll('img')
 
   imageLoaded: (iter) ->
-    return unless @isMounted()
```

Comment:
I'm guessing this was added to avoid the warning from the @setState below if the image loads after the component is unmounted... is it ok to remove? Unlike the other files I see nothing that removes the listener on unmount...